.uberenv_config.json

```diff
@@ -4,7 +4,7 @@
 "package_final_phase": "initconfig",
 "package_source_dir": "../..",
 "spack_url": "https://github.com/spack/spack.git",
-"spack_branch": "e4s-23.02",
+"spack_branch": "v0.20.1",
```

Comment:
We are now using "develop-2024-01-21".

Reply:
Fixed in https://github.com/LLNL/CARE/pull/246